### ISSUE TITLE
missing dm3.map

### ISSUE THREAD

**Issue by jxsl13** [ZillyInsta/maps#1]
there is a discrepancy between 0.6 and 0.7 maps.
0.6 is misisng dm3.map.
might make sense to either remove it in 0.7 maps or add it here.
dunno when dm3 was introduced, might be a 0.7 map only?

**Comment by ChillerDragon** [comment]
Yes dm3 was added in 0.7 https://github.com/teeworlds/teeworlds-maps/commits/master/dm3.map

So I have to port it to 0.6
